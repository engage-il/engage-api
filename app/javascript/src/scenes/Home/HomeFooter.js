// @flow
import * as React from 'react'
import styled from 'react-emotion'
import { Footer } from '@/components'
import { mixins, colors } from '@/styles'
import { quill } from '&/images'

const size = {
  width: 77,
  height: 25
}

export function HomeFooter () {
  return (
    <Footer>
      <Background>
        <Content>
          <img
            src={quill}
            alt='Quill Logo'
            style={size}
            {...size}
          />
          <h3>Non-partisan, educational, accessible</h3>
          <p>Legislated is a non-profit organization started at Chi Hack Night by a dedicated group of volunteers.</p>
        </Content>
      </Background>
    </Footer>
  )
}

const Background = styled.div`
  ${mixins.flexRow};
  ${mixins.flexCenter};

  padding: 50px 25px;
  background-color: ${colors.secondary};
`

const Content = styled.div`
  ${mixins.flexColumn};

  align-items: center;
  max-width: 550px;
  color: ${colors.white};
  text-align: center;

  > * + * {
    margin-top: 15px;
  }
`
