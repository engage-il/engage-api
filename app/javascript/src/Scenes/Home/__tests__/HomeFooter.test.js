/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import { HomeFooter } from '../HomeFooter'

// subject
let subject

function loadSubject () {
  subject = shallow(<HomeFooter />)
}

// specs
afterEach(() => {
  subject = null
})

describe('#render', () => {
  it('shows the home footer', () => {
    loadSubject()
    expect(subject).toMatchSnapshot()
  })
})
