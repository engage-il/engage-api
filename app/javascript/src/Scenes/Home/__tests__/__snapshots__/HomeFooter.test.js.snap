// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render shows the home footer 1`] = `
<Footer>
  <Background>
    <Content>
      <img
        alt="Quill Logo"
        height={25}
        src="http://website.com/image.png"
        style={
          Object {
            "height": 25,
            "width": 77,
          }
        }
        width={77}
      />
      <h3>
        Non-partisan, educational, accessible
      </h3>
      <p>
        Legislated is a non-profit organization started at Chi Hack Night by a dedicated group of volunteers.
      </p>
    </Content>
  </Background>
</Footer>
`;
